modules/nf-core/smoove/meta.yml

```
@@ -0,0 +1,56 @@
+name: "smoove"
+description: smoove simplifies and speeds calling and genotyping SVs for short reads. It also improves specificity by removing many spurious alignment signals that are indicative of low-level noise and often contribute to spurious calls. Developped by Brent Pedersen.
+keywords:
+  - structural variants
+  - SV
+  - vcf
+  - wgs
+tools:
+  - "smoove":
+      description: "structural variant calling and genotyping with existing tools, but, smoothly"
+      homepage: https://github.com/brentp/smoove
+      documentation: https://brentp.github.io/post/smoove/
+      tool_dev_url: https://github.com/brentp/smoove
+      doi: ""
+      licence: "['Apache-2.0']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - bam:
+      type: file
+      description: BAM file
+      pattern: "*.{bam}"
+  - index:
```

```suggestion
  - bai:
```

You should use the same names as in the module itself for clarity

---

changed. Thanks